Neos.Neos/Documentation/References/PropertyEditorReference.rst

```diff
@@ -400,6 +400,35 @@ Options Reference:
 ``threshold`` (number)
 	Minimum amount of characters which trigger a search. Default is set to 2.
 
+``createNew`` (array)
+    If you are using the rewritten Neos Ui, it is also possible to create new selectable nodes directly from the reference editor.
+    This can come in handy for example if you reference tag nodes and want to add new categories on the fly.
```

Comment:
You first speak about tag nodes, then about categories. Since both is commonly used in a different way, I would go for either tag or category.